docs/src/domains/quick_start.md

```diff
@@ -63,3 +68,8 @@ Here's what will now happen under the hood.  Assume for a moment that all caches
 6. Your browser sends a request to `1.2.3.4` for domain `www.mysite.com`.  Your router passes the request through to your application which in turn responds with whatever it's supposed to do.
 
 On subsequent requests, your browser will know to simply connect to `1.2.3.4` for domain `www.mysite.com` and skip the rest.  The entire process takes only a few milliseconds.
+
+{{< note >}}
```

Comment:
I think Larry meant for this to be somewhere in the "Set domains" section.

Reply:
Thank you @chadwcarlson